site/documentation/content/admin-guide/device-connection-config.md

```diff
@@ -90,19 +90,45 @@ The server may be configured to open both a secure and a non-secure port at the
 | :------------------------------------------ | :-------: | :------ | :-----------------------------------------------------------------------|
-| `HONO_DEVICECONNECTION_REMOTE_SERVER_LIST`<br>`--hono.deviceConnection.remote.serverList` | yes | - | A list of remote servers in the form: `host1[:port][;host2[:port]]....`. |
-| `HONO_DEVICECONNECTION_REMOTE_AUTH_SERVER_NAME`<br>`--hono.deviceConnection.remote.authServerName` | yes | - | The server name to indicate in the SASL handshake when authenticating to the server. |
-| `HONO_DEVICECONNECTION_REMOTE_AUTH_USERNAME`<br>`--hono.deviceConnection.remote.authUsername` | yes | - | The username to use for authenticating to the server. |
-| `HONO_DEVICECONNECTION_REMOTE_AUTH_PASSWORD`<br>`--hono.deviceConnection.remote.authPassword` | yes | - | The password to use for authenticating to the server. |
+| `HONO_DEVICECONNECTION_REMOTE_SERVERLIST`<br>`--hono.deviceConnection.remote.serverList` | yes | - | A list of remote servers in the form: `host1[:port][;host2[:port]]....`. |
+| `HONO_DEVICECONNECTION_REMOTE_AUTHSERVERNAME`<br>`--hono.deviceConnection.remote.authServerName` | yes | - | The server name to indicate in the SASL handshake when authenticating to the server. |
+| `HONO_DEVICECONNECTION_REMOTE_AUTHREALM`<br>`--hono.deviceConnection.remote.authRealm` | yes | - | The authentication realm for the SASL handshake when authenticating to the server. |
+| `HONO_DEVICECONNECTION_REMOTE_AUTHUSERNAME`<br>`--hono.deviceConnection.remote.authUsername` | yes | - | The username to use for authenticating to the server. |
+| `HONO_DEVICECONNECTION_REMOTE_AUTHPASSWORD`<br>`--hono.deviceConnection.remote.authPassword` | yes | - | The password to use for authenticating to the server. |
 
 In general, the service supports all configuration properties of the [Infinispan Hotrod client](https://docs.jboss.org/infinispan/9.4/apidocs/org/infinispan/client/hotrod/configuration/package-summary.html#package.description) using `hono.deviceConnection.remote` instead of the `infinispan.client.hotrod` prefix.
 
+### Embedded cache
+
+The following table provides an overview of the configuration variables and corresponding command line options for configuring the embedded cache:
+
+hono.device-connection.embedded.configuration-file
+
+| Environment Variable<br>Command Line Option | Mandatory | Default | Description                                                             |
+| :------------------------------------------ | :-------: | :------ | :-----------------------------------------------------------------------|
+| `HONO_DEVICECONNECTION_EMBEDDED_CONFIGURATIONFILE`<br>`--hono.deviceConnection.embedded.configurationFile` | no | - | The absolute path to an Infinispan configuration file. Also see the[Infinispan Configuration Schema](https://docs.jboss.org/infinispan/9.4/configdocs/). |
```

Comment:
don't want to be picky but IMHO there should be space between *Also see the* and the link ...

Reply:
No worries, I couldn't agree more! :)